tests/mentorship_relation/test_api_list_relations.py

```diff
@@ -199,7 +199,7 @@ def test_list_filter_mentorship_relations_empty(self):
     def test_list_filter_mentorship_relations_accepted(self):
         with self.client:
             response = self.client.get(
-                "/mentorship_relations?relation_state=%s" % "accepted",
+                f"/mentorship_relations?relation_state={'accepted'}",
```

Comment:
```suggestion
                "/mentorship_relations?relation_state=accepted",
```
No need for f-strings when you don't have a variable and is just a string :) Can you please fix this line and the rest I will mention